src/PhpSpec/Formatter/JUnitFormatter.php

```diff
@@ -52,7 +52,7 @@ class JUnitFormatter extends BasicFormatter
         ExampleEvent::SKIPPED => 'skipped',
     );
 
-    public function __construct(PresenterInterface $presenter, IO $io, StatisticsCollector $stats)
+    public function __construct(Presenter $presenter, IO $io, StatisticsCollector $stats)
```

Comment:
technically, all those typehints must be changed in 2.x already. Otherwise, it is impossible to avoid using the BC layers (as everything forces you to rely on the deprecated interface)